src/cubecl_sum_udaf.rs

```diff
@@ -90,8 +90,8 @@ impl<R: Runtime> AggregateUDFImpl for GpuSum<R> {
 
 struct GpuSumAccumulator<R: Runtime> {
     compute_client: Arc<ComputeClient<R::Server, R::Channel>>,
-    // TODO: Having a generic number here is very difficult, I still don't know how to do it, so I just use f32.
     result: f32,
```

Comment:
It seems like this comment is still relevant, it would be nice to keep it for future readers:

```suggestion
    // TODO: Having a generic number here is very difficult, I still don't know how to do it, so I just use f32.
    result: f32,
```